### PR TITLE
Update on how to handle properties with slashes as names with a human readable example

### LINKED ISSUE #2505: Update Documentation on how to handle properties with slashes as names with a human readable example
### Component

None

### Expected Behavior
Documentation contains information on how to handle properties with slashes as names with a human readable example like:
```
          <resources>
              <annotations>
                <service>
                  <property>
                    <name>cloud.google.com/neg</name>
                    <value>{"ingress":true}</value>
                  </property>
                </service>
              </annotations>
          </resources>
        </configuration>
```

### Acceptance Criteria
- [ ] Documentation is updated

### PR DESCRIPTION
**Pull request by DominikBasnerSotec**
Adding information on annotations with special characters

## Description
This PR updates the documentation to provide information on special characters in kubernetes annotations

[Documentation preview](https://ci.eclipse.org/jkube/job/Build/job/Documentation/job/PR-2506/3/artifact/docs-generated/kubernetes-maven-plugin.html#xml-configuration)

Fixes #2505

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.dev/jkube/contributing)
 - [ ] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] My code follows the style guidelines of this project
 - [x] I Keep It Small and Simple: The smaller the PR is, the easier it is to review and have it merged
 - [ ] I use [conventional commits](https://www.conventionalcommits.org/) in my commit messages
 - [ ] I have performed a self-review of my code
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [x] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by manusa** [comment]
### Eclipse JKube [CI Report](https://github.com/jkubeio/ci)

Started new GH workflow run for https://github.com/eclipse/jkube/pull/2506 (_2023-12-21T14:07:00Z_)

:gear: [JKube E2E Tests (7289160343)](https://github.com/jkubeio/ci/actions/runs/7289160343) :hourglass_flowing_sand:
- :hourglass_flowing_sand: [Bootstrap CI test run (#2506)](https://github.com/jkubeio/ci/actions/runs/7289160343/job/19863220802)

**Comment by rohanKanojia** [comment]
Welcome, Thanks a lot for contributing to Eclipse JKube :+1:,

In order to contribute to Eclipse, you would need to create an [Eclipse Account](https://accounts.eclipse.org/) and Sign [Eclipse Contributor Agreement](https://accounts.eclipse.org/user/login?destination=user/eca). Once you've created an eclipse account you would need to update your commit message with a `Signed-off-by:` footer. You can do it like this:
```
# (Optional): Configure Git User and Email
git config user.name "Your Name"
git config user.email "your_email@example.com"

# Amend your previous commit with signed off footer
git commit --amend --signoff
```

File: kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_introduction.adoc
Comment (rohanKanojia) [inline kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_introduction.adoc:320]: Do you think [Labels And Annotations section](https://eclipse.dev/jkube/docs/kubernetes-maven-plugin/#labels-annotation-configuration) would be a more appropriate place for this?

**Review by rohanKanojia — APPROVED** [review]

**Comment by DominikBasnerSotec** [comment]
@manusa @sunix @rohanKanojia I can't merge, since I don't have write access.

**Comment by rohanKanojia** [comment]
@DominikBasnerSotec : We usually require at least 2 approvals before merging pull requests. You would need to wait until someone else on the team also approves it.

**Review by manusa — APPROVED** [review]
Awesome, thx! :rocket:

**Pull request merged**
